pkg/plugin/connector/builtin/registry.go

```diff
@@ -63,7 +65,7 @@ type blueprint struct {
 
 type DispenserFactory func(name plugin.FullName, logger log.CtxLogger) connector.Dispenser
```

Comment:
I think anything related to `DispenserFactory` could be made private now, right?

Reply:
Yup!